src/net_processing.cpp

```diff
@@ -413,7 +413,7 @@ class PeerManagerImpl final : public PeerManager
     /** Pointer to this node's banman. May be nullptr - check existence before dereferencing. */
     BanMan* const m_banman;
     ChainstateManager& m_chainman;
-    CTxMemPool& m_mempool;
+    CTxMemPool* m_mempool;
```

Comment:
```suggestion
    CTxMemPool* const m_mempool;
```

Could keep this const to enforce the assumption that the pointer can't be changed

Reply:
Done!